bobtemplates/plone/content_type/profiles/default/types/+dexterity_type_name_fti+.xml.bob

```diff
@@ -21,6 +21,7 @@
 {{% endif %}}
   <property name="link_target"></property>
 
+{{% if  dexterity_type_base_class == 'Container' %}}
   <!-- Hierarchy control -->
   <property name="global_allow">{{{ dexterity_type_global_allow }}}</property>
```

Comment:
why are you want to remove the global_allow property here?
Please read the issue describtion, it's all in there.